pandas/tests/io/formats/style/test_format.py

```diff
@@ -305,3 +305,20 @@ def test_precision_zero(df):
     ctx = styler._translate(True, True)
     assert ctx["body"][0][2]["display_value"] == "-1"
     assert ctx["body"][1][2]["display_value"] == "-1"
+
+
+def test_format_options_validator():
```

Comment:
Nitpick but this could easily be parametrized and use the follow for the cases that raise/don't raise. https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising

Reply:
yeh normally I do do this, if this pr is left open for a few days will get to it